src/editor/autocodecompleter.cpp

```diff
@@ -1,49 +1,69 @@
 #include "autocodecompleter.h"
 #include<QDebug>
+#include"keypressevents.h"
 
 AutoCodeCompleter::AutoCodeCompleter(const QStringList &completions, QObject *parent):
     QCompleter(completions, parent)
 {
     connect(this, SIGNAL(activated(QString)), this, SLOT(replaceCurrentWord(QString)));
```

Comment:
@PeterLysyk
Used old connection
In new style:
**connect(this, &AutoCodeCompleter::activated,
                 this, &AutoCodeCompleter::replaceCurrentWord,
                 Qt::AutoConnection);**